src/literature/components/GamePage/index.js

```diff
@@ -183,9 +183,15 @@ const GamePage = (props) => {
 		setAbandonOpen(false)
 	}
 
+	const handleClickErrorLeave = () => {
+		let response = {
```

Comment:
You don't have to send a response for leaveGameSuccess. Just calling the function should be enough.